### PR TITLE
Show last edited by on information pages

### LINKED ISSUE #54: Add "last edited by" for Information Pages
Currently, only the date of the latest edition of a page is displayed. Also show the author of the latest edition.

### PR DESCRIPTION
**Pull request by frcroth**
Closes #54

### CODE REVIEW

**Comment by jeriox** [comment]
This always shows the author, in the issue we talked about only showing it sometimes. In 1327 you have a setting per page to control this

**Comment by frcroth** [comment]
> This always shows the author, in the issue we talked about only showing it sometimes. In 1327 you have a setting per page to control this

That was not entirely clear to me from the issue. Should it also be the choice between showing the author to None, Logged in and Everyone?

**Comment by jeriox** [comment]
I think I`d also be fine with a checkbox "show author", I can't really see the necessity for the "logged in" option

**Review by jeriox — APPROVED** [review]

File: myhpi/core/models.py [hunk 1]
```diff
@@ -31,6 +33,10 @@ class InformationPage(Page):
         "RootPage",
     ]
 
+    @property
+    def last_edited_by(self):
+        return self.get_latest_revision().user
```
Comment (SilvanVerhoeven) [inline myhpi/core/models.py:38]: @frcroth Where did you find this function? Even now I simply cannot find it in the [reference](https://docs.wagtail.org/en/stable/reference/pages/model_reference.html#page)

Comment (frcroth) [reply myhpi/core/models.py:38]: https://github.com/wagtail/wagtail/blob/d641f6a9ac4e79b5451d4c366a2f95ab78be1769/wagtail/models/__init__.py#L1004

**Pull request merged**